flocker/node/agents/blockdevice.py

```diff
@@ -1147,10 +1147,13 @@ def __init__(self, root_path, compute_instance_id):
             different ``compute_instance_id``.
```

Comment:
:question: Maybe document that one will now be created if the argument is omitted.